### DISCUSSION TITLE
Contributing Guide

### DISCUSSION THREAD

**Discussion by akamfoad** [Code-Forge-Net/Remix-Forge#8] [Ideas]
I'm a huge fan of Remix and the forge ecosystem looks very promising, I'd love to contribute to this project, but I find it a little hard to do that. I think having a contribution guide can help a lot, things like how to build and install the extension locally and etc.

Having a guide like this helps getting people running fast and makes contribution much easier in my opinion.

Thanks for the amazing work that has been done so far btw <3

**Comment by AlemTuzlak** [comment]
Yeah unfortunately it's only been me working on 3 oss libraries and I don't have a lot of free time so prioritization has been mostly on getting features out there. I will be adding a contribution guide shortly and making it a bit easier to contribute in the future. Thank you so much.

**Reply by akamfoad** [reply]
Amazing, I loved the dev tools and the fact that it can be integrated with VSCode. You have done great work, looking forward to those guides!

**Comment by AlemTuzlak** [comment]
@akamfoad done!
